### PR TITLE
added Fairing - Network CPU & GPU

### PR DESCRIPTION
**Pull request by Edwardsleo**

### CODE REVIEW

File: apps/networking/network-traffic/onprem/fairing/Network-Classification-fairing.ipynb [hunk 1]
```diff
@@ -0,0 +1,2380 @@
+{
```
Comment (swiftdiaries) [inline apps/networking/network-traffic/onprem/fairing/Network-Classification-fairing.ipynb:1]: <p>Should we clear outputs before checking in this commit?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CiscoAI/cisco-kubeflow-starter-pack/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (swiftdiaries) [inline apps/networking/network-traffic/onprem/fairing/Network-Classification-fairing.ipynb:1]: <p>Can we add a version tag for <code>kubeflow-fairing</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CiscoAI/cisco-kubeflow-starter-pack/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (swiftdiaries) [inline apps/networking/network-traffic/onprem/fairing/Network-Classification-fairing.ipynb:1]: <p>Can we move <code>DOCKER_REGISTRY</code>  to the top of the notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CiscoAI/cisco-kubeflow-starter-pack/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (swiftdiaries) [inline apps/networking/network-traffic/onprem/fairing/Network-Classification-fairing.ipynb:1]: <p>Can we move <code>device_type</code>  to the top of the notebook?</p><p>The idea being that all configuration is done at the start of the notebook and the rest of the steps can be run without much changes</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CiscoAI/cisco-kubeflow-starter-pack/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (swiftdiaries) [inline apps/networking/network-traffic/onprem/fairing/Network-Classification-fairing.ipynb:1]: <p>Clean up is really neat :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CiscoAI/cisco-kubeflow-starter-pack/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Comment (Edwardsleo) [reply apps/networking/network-traffic/onprem/fairing/Network-Classification-fairing.ipynb:1]: > Should we clear outputs before checking in this commit?
> 
> 
> _Reply via [ReviewNB](https://app.reviewnb.com/CiscoAI/cisco-kubeflow-starter-pack/pull/66/discussion/)_

Cleared Credentials cell alone.

Comment (Edwardsleo) [reply apps/networking/network-traffic/onprem/fairing/Network-Classification-fairing.ipynb:1]: > Can we add a version tag for `kubeflow-fairing`?
> 
> 
> _Reply via [ReviewNB](https://app.reviewnb.com/CiscoAI/cisco-kubeflow-starter-pack/pull/66/discussion/)_

added version tag in Fairing-ReadME

Comment (Edwardsleo) [reply apps/networking/network-traffic/onprem/fairing/Network-Classification-fairing.ipynb:1]: > Can we move `DOCKER_REGISTRY` to the top of the notebook?
> 
> 
> _Reply via [ReviewNB](https://app.reviewnb.com/CiscoAI/cisco-kubeflow-starter-pack/pull/66/discussion/)_

addressed this issue..all config details will be on top

Comment (Edwardsleo) [reply apps/networking/network-traffic/onprem/fairing/Network-Classification-fairing.ipynb:1]: > Can we move `device_type` to the top of the notebook?
> 
> The idea being that all configuration is done at the start of the notebook and the rest of the steps can be run without much changes
> 
> 
> _Reply via [ReviewNB](https://app.reviewnb.com/CiscoAI/cisco-kubeflow-starter-pack/pull/66/discussion/)_

addressed this issue

**Pull request merged**